docs/dev/annual_updates.rst

```diff
@@ -197,6 +197,12 @@ to the existing coding tables.
 
 Run a Siloed EIA ETL
 --------------------
+.. note::
```

Comment:
Seems like we should go ahead and make this update now?

Reply:
did you seem my comment about this? i'd like to wait to actually update this when I am able to actually run a batch of new ferc1 utilities through the process.

Reply:
Sorry no I didn't see it when I was over in the files view.